python/lsst/ts/xml/data/sal_interfaces/ATMonochromator/ATMonochromator_Commands.xml

```diff
@@ -7,9 +7,9 @@
     <Description>Command to change the wavelength of the monochromator</Description>
     <item>
       <EFDB_Name>wavelength</EFDB_Name>
-      <Description>wavelength</Description>
+      <Description>wavelength to be calibrated</Description>
       <IDL_Type>float</IDL_Type>
-      <Units>meter</Units>
+      <Units>nanometer</Units>
```

Comment:
Default astropy unit is nm (also used below) -> consider change that to nm?
```
In [1]: import astropy.units as u

In [2]: u.nm
Out[2]: Unit("nm")

In [3]: u.nanometer
Out[3]: Unit("nm")
```

Reply:
Also changes seconds to s and millimeters to m